### PR TITLE
Little adjustments nick wants

### PR DESCRIPTION
**Pull request by AriadneLovelace**
- Change title
- Change about writing
- Add scroll bar to carousel
- Make scroll bar bigger and visible

### CODE REVIEW

File: Dashboard/Views/MainWindowView.axaml [hunk 1]
```diff
@@ -22,7 +22,7 @@
             <Grid ColumnDefinitions="0.8*, 0.2*" Margin="10, 0">
                 <TextBlock Grid.Column="0" FontWeight="Black" FontFeatures="Bold"
                            FontSize="25" VerticalAlignment="Center">
-                    Nuvotion FSAE-Scrutineering Tool
+                    Formula Autonomous Vehicle Scrutineering Tool
```
Comment (nrdag) [inline Dashboard/Views/MainWindowView.axaml:25]: Might want to change the window title too?

File: Dashboard/Views/AboutView.axaml [hunk 2]
```diff
@@ -39,7 +39,9 @@
                     Margin="0 10">
                 <StackPanel Spacing="5">
                     <TextBlock Name="IntroParagraph" FontSize="18">
-                        This program was developed for Nuvotion Pty Ltd. during an RMIT University Capstone project.
+                        Developed for Nuvotion as part of the RMIT University Software Capstone Project 2024.
```
Comment (nrdag) [inline Dashboard/Views/AboutView.axaml]:
```suggestion
                        Developed for Nuvotion as part of the RMIT University Software Engineering Capstone Project 2024.
```

File: Dashboard/Views/ScrutineeringView.axaml [hunk 3]
```diff
@@ -76,87 +77,89 @@
...
-                                                                </MultiBinding>
-                                                            </TextBlock.Text>
-                                                        </TextBlock>
-                                                    </ControlTemplate>
-                                                </ContentControl.Template>
-                                            </ContentControl>
-                                        </DataTemplate>
-                                    </ItemsControl.ItemTemplate>
-                                </ItemsControl>
+                <ScrollViewer VerticalScrollBarVisibility="Auto" AllowAutoHide="False">
```
Comment (nrdag) [inline Dashboard/Views/ScrutineeringView.axaml:80]: This overlaps the next button on smaller screens, if that matters...
image

Comment (AriadneLovelace) [reply Dashboard/Views/ScrutineeringView.axaml:80]: I cant change where the buttons go for the carousel neither with the scroll, i showed it to nick and he said "that is ace!" so i think its okay

File: Dashboard/Views/MainWindowView.axaml (hunk 1 above)
Comment (AriadneLovelace) [reply Dashboard/Views/MainWindowView.axaml:25]: 25e7e8b17b8cb3e65cf1fcefa4278b5b7514f39a

File: Dashboard/Views/AboutView.axaml (hunk 2 above)
Comment (AriadneLovelace) [reply Dashboard/Views/AboutView.axaml]: 25e7e8b17b8cb3e65cf1fcefa4278b5b7514f39a

**Comment by S3784866** [comment]
LGTM.

**Pull request merged**